backend/tests/app/business/open_food_facts/test_knowledge_panel.py

```diff
@@ -18,22 +17,14 @@
     BreedingTypeAndWeight,
     KnowledgePanelResponse,
 )
+from tests.app.business.open_food_facts.mocks import product_data
 
 
 @pytest.mark.asyncio
 async def test_get_data_from_off_success():
```

Comment:
You can use the fixture like that:

```suggestion
async def test_get_data_from_off_success(product_data: ProductData):
```